### PR TITLE
Add PPO documentation

### PR DESCRIPTION
**Pull request by vwxyzjn**
## Types of changes
- [x] Documentation

## Checklist:

- [x] I've read the [CONTRIBUTION](https://github.com/vwxyzjn/cleanrl/blob/master/CONTRIBUTING.md) guide (**required**).
- [x] I have ensured `pre-commit run --all-files` passes (**required**).
- [x] I have updated the documentation and previewed the changes via `mkdocs serve`.
- [x] I have updated the tests accordingly (if applicable).

If you are adding new algorithms or your change could result in performance difference, you may need to (re-)run tracked experiments. See https://github.com/vwxyzjn/cleanrl/pull/137 as an example PR. 
- [x] I have contacted @vwxyzjn to obtain access to the [openrlbenchmark W&B team](https://wandb.ai/openrlbenchmark) (**required**).
- [x] I have tracked applicable experiments in [openrlbenchmark/cleanrl](https://wandb.ai/openrlbenchmark/cleanrl) with `--capture-video` flag toggled on (**required**).
- [x] I have added additional documentation and previewed the changes via `mkdocs serve`.
    - [x] I have explained note-worthy implementation details.
    - [x] I have explained the logged metrics.
    - [x] I have added links to the original paper and related papers (if applicable).
    - [ ] I have added links to the PR related to the algorithm.
    - [x] I have created a table comparing my results against those from reputable sources (i.e., the original paper or other reference implementation).
    - [x] I have added the learning curves (in PNG format with `width=500` and `height=300`).
    - [x] I have added links to the tracked experiments.
- [x] I have updated the tests accordingly (if applicable).

### CODE REVIEW

**Comment by vwxyzjn** [comment]
@yooceii @dosssman this is ready for review. I didn't do the `I have added links to the PR related to the algorithm` because I found adding the PR to be less helpful than I anticipated... Ideally, the users should be able to reproduce the same results in the latest master without checking out each PR.

File: docs/rl-algorithms/ppo.md [hunk 1]
```diff
@@ -88,12 +110,14 @@ Learning curves:
 
 Tracked experiments and game play videos:
```
Comment (yooceii) [inline docs/rl-algorithms/ppo.md:111]: nit, if it's classic control env, then probably use videos instead game play videos.

**Review by yooceii — APPROVED** [review]
lgtm

**Review by dosssman — APPROVED** [review]
All clear on my side too.

**Pull request merged**